src/lib/storage/migration/1567674609659-reencrypt-seed.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * This migration re-encrypts the encryptedEntropy in the wallet database using
+ * the jolocom-lib (which uses node crypto) instead of the previously used
+ * crypto-js dependency
+ */
+
+import { MigrationInterface, QueryRunner } from 'typeorm/browser'
+import { reencryptWithJolocomLib } from 'src/lib/compat/cryptojs'
+
+async function getPassword(): Promise<string> {
+  let password: string | undefined
+  // at the time of writing this file, the encryption password is stored in
+  // react-native-keychain and accessesed through src/lib/keychain.ts, but code
+  // is reproduced here in case things change in the future
+
+  if (typeof navigator !== 'undefined' && navigator.product == 'ReactNative') {
+    const Keychain = require('react-native-keychain')
+    password = await Keychain.getGenericPassword()
+    if (!password) {
+      throw new Error(
+        "Can't load password from react-native-keychain. Got: " + password
```

Comment:
This is only executed if `!password`. The `+ password` add the end is adding nothing!?

Reply:
Yeah I left it there to say "null" or "undefined" or whatever, just thought it wouldn't hurt :D